WooCommerce/WooCommerceTests/ViewRelated/Blaze/BlazeCampaignListViewModelTests.swift

```diff
@@ -456,4 +456,10 @@ private extension BlazeCampaignListViewModelTests {
         }
         storage.saveIfNeeded()
     }
+
+    func insertProduct(_ readOnlyProduct: Product) {
+        let newProduct = storage.insertNewObject(ofType: StorageProduct.self)
+        newProduct.update(with: readOnlyProduct)
+        storage.saveIfNeeded()
+    }
```

Comment:
Is this being used in this file?

Reply:
Deleted in https://github.com/woocommerce/woocommerce-ios/pull/11671/commits/21223a1f937eb0e9334d8958975960f40220b445

This was missed when I deleted unit tests from this file. Reminder about unused functions / vars is something I wish Xcode would copy from Android Studio 😅